drive/client.py

```diff
@@ -351,7 +351,7 @@ def download_file(self, file_id: str, path: str, mime_type: Optional[str] = None
         with open(path, "wb") as f:
             self.download(file_id, f, mime_type=mime_type)
 
-    def download_excel_workbook(self, file_id: str, read_only=False):
+    def download_excel_workbook(self, file_id: str, read_only: bool = False) -> openpyxl.Workbook:
```

Comment:
```suggestion
    def download_excel_workbook(self, file_id: str, read_only = False) -> openpyxl.Workbook:
```
see other similar comments above

Reply:
See above: https://github.com/bfontaine/drive/pull/13#discussion_r2015341970